### DISCUSSION TITLE
project doc website: sidebar obscures the text, can't be hidden

### DISCUSSION THREAD

**Discussion by CouldBeThis** [advplyr/audiobookshelf#3581] [General]
Hello, there is a problem with the project's documentation website. The left sidebar covers a lot of the text. There isn't a horizontal scrollbar by which you can even awkwardly uncover it. There is no hide button for the sidebar.

Likely, whoever designed this has a much larger display and/or is comfortable with smaller text. (I have a 12" notebook.) While generally it's good for design to be flexible for a variety of text sizes, audiobooks are *specifically* something that blind or low-vision people are interested in so so definitely be accommodated.

The layout is somewhat responsive as in the sidebar will hide when viewed on mobile or if you make the window *very* small. 

Examples from this URL: https://www.audiobookshelf.org/docs#linux-install-deb

Here is how this page looks maximized at a comfortable font size for me to read:

![image](https://github.com/user-attachments/assets/fb321c4b-556d-47a6-b490-65675fcc3b6f)

With the window maximized, this is the largest font size I can use to see the entire content:

![2024-11-02_13-47](https://github.com/user-attachments/assets/4c982039-febc-41a1-827c-665ed0e70458)

The IRL x-height of the default body font as it renders on my display is just under 2mm. It's too small. 

I guess I could use u block origin to zap the sidebar, but then I wouldn't have the table of contents available. 

Firefox 133 on linux; used private browsing with no extensions enabled with same results.

**Comment by nichwall** [comment]
The website repo is here
https://github.com/audiobookshelf/audiobookshelf-web

**Reply by CouldBeThis** [reply]
thanks! posted to https://github.com/audiobookshelf/audiobookshelf-web/issues/111
